package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.1",
   "description": "Mailinator REST API client for JavaScript applications.",
   "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "types": "types/index.d.ts",
   "files": [
     "lib/**/*"
```

Comment:
I think you need to also add "types" to the array of files for the types folder to be published on npm.
Otherwise, the types don't really need to be in a separate folder, you could also just remove the "declarationDir" line in tsconfig.json and not commit the type definitions. They'll get generated at build time in the lib folder. Really the only change that are needed to fix this issue is adding `"declaration": true` and `"skipLibCheck": true` to the tsconfig.json.

Reply:
Thanks for the suggestions.  I'd make the change, except it seems like this is a dead project.  I'd recommend that someone who knows what they are doing should add the types to Definitely Typed.

Reply:
Hello guys. I'm so sorry for the late response, but I've just got access to this repository. It's not initially my code, but as far as I have access I'll merge and close this pull request. 
Feel free to let me know if you have new issues or improvements.

Thanks, 
Marian